bin/pygrb/pycbc_grb_trig_combiner

```diff
@@ -54,68 +53,13 @@ TQDM_KW = {
 
 # -- utilties -----------------------------------
 
-def _init_combined_file(h5file, attributes, datasets, **compression_kw):
-    h5file.attrs.update(attributes)
-    # create datasets
-    for dset, (shape, dtype) in datasets.items():
-        if "template_id" in dset:
-            continue
-        h5file.create_dataset(dset, shape=shape, dtype=dtype,
-                              **compression_kw)
-
-
-def _merge_dataset(source, target, position, is_event_id=False, ifo_event_id_pos=None):
-    """Merge one dataset into another at a given position.
-
-    Parameters
-    ----------
-    source : `h5py.Dataset`
-        the source dataset
-
-    target : `h5py.Dataset`
-        the target dataset
-
-    position: `int`
-        the index of `target` at which to insert ``source`
-
-    is_event_id : `bool`, optional
-        if `True` the values of ``source`` will be incremented by ``position``;
-        this should be used to increment the ``*/event_id`` columns by the
-        internal count of events
-
-    ifo_event_id_pos : `int`, `None`, optional
-        if not `None`, the values of ``source`` will be incremented by
-        the given value; this should be used to increment the
-        ``network/X1_event_id`` column values by the current count of
-        ``X1/event_id`` events
-
-    Returns
-    -------
-    size : `int`
-        the size of the dataset that was just merged
-    """
-    data = source[()]
-    size = data.shape[0]
-    # increment network/X1_event_id by count of X1/event_id
-    if ifo_event_id_pos:
-        data += ifo_event_id_pos
-    # increment X1/event_id by number of events recorded so far
-    elif is_event_id:
-        data += position
-    target[position:position+size] = data
-    return size
-
-
 def merge_hdf5_files(inputfiles, outputfile, verbose=False, **compression_kw):
     """Merge several HDF5 files into a single file
 
     Parameters
     ----------
     inputfiles : `list` of `str`
         the paths of the input HDF5 files to merge
-
-    outputfile : `str`
-        the path of the output HDF5 file to write
```

Comment:
The `outputfile` argument is still present, so you should keep this.

Reply:
Right, it's not the output item (which I removed as it was unused).